src/Gizmos/gizmo.ts

```diff
@@ -321,6 +322,25 @@ export class Gizmo implements IDisposable {
         }
     }
 
+    /**
+     * refresh gizmo mesh material depending on state of dragBehavior
+     * @param enabledDragBehavior drag behavior is enabled or not
+     * @param enableddMaterial material to apply if drag behavior is enabled
+     * @param disableMaterial material to apply if drag behavior is disenabled
+     */
+    protected _refreshForDragBehavior(enabledDragBehavior: boolean, enabledMaterial: StandardMaterial, disableMaterial: StandardMaterial) {

```

Comment:
if `enabledDragBehavior` is only used to check which material is being used, why not only pass the material that is going to be used in the first place?

Reply:
I believe this is used to switch the materials on enabled vs not enabled. if enabled=true, colored material. If not, gray material

Reply:
The three variables passed to the function are only being used by themselves. a boolean and two materials are passed to get a material that could have been passed on its own in the first place.

Reply:
yep true, would probably be cleaner to just pass in the material itself and maybe rename to setMaterial or sth